CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## Release 0.6.0
+
+New features:
+- Preliminary work to add capabilities to nodes, so that the UI will
+  be able to send actions down to the probe.
+  [#315](https://github.com/weaveworks/scope/issues/315)
+- Added support for pcap based packet sniffing, to provide bandwidth
```

Comment:
I'd potentially add a new section "experimental features" and provide more information on how to enable it, and what to expect.

Or ditch it.